.nvmrc

```diff
@@ -0,0 +1 @@
+20.17.0
```

Comment:
Whilst you're doing this can you bump to the node version. At the time of writing it's currently `v22.12.0`. Remember to bump in the node package json file for the node engine version as well.